### PR TITLE
Implement the genre class

### PR DESCRIPTION
**Pull request by chimwemwe007**
### The following was completed in this branch for this this pull request.

1. add_item method in the Genre class
2. should take an instance of the Item class as an input
3. should add the input item to the collection of items
4. should add self as a property of the item object (by using the correct setter from the item object)

### CODE REVIEW

File: classes/genre.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  def initialize(name)
+    @id = Random.rand(1...1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(album)
+    @items << album
+    album.genre = self
```
Comment (Ney2) [inline classes/genre.rb:14]: As shown in the diagram kindly change the genre method to add_genre

File: classes/genre.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  private
+
+  attr_reader :id, :items
```
Comment (Ney2) [inline classes/genre.rb:19]: Since items are used in the add_item method it is better to change the attr_reader to attr_accessor

**Review by Ney2 — CHANGES_REQUESTED** [review]
Hi @chimwemwe007  😃, 👋,

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- The repo looks professional ✔️ 
-  Your code is clean ✔️ 
- No Linter errors ✔️ 

## Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏
Please feel free to tag my name and ask questions if there is anything that is not clear.

File: classes/genre.rb (hunk 2 above)
Comment (chimwemwe007) [reply classes/genre.rb:19]: alright will do

File: classes/genre.rb (hunk 1 above)
Comment (chimwemwe007) [reply classes/genre.rb:14]: Alright working on it.

**Review by Ney2 — APPROVED** [review]
## Staus Approved ✔️ ✔️ ✔️ 
Good job, 💯 There is nothing else to say than it is time to merge it



- Cheers and happy coding 👏 👏 👏

**Pull request merged**